ipa-core/src/protocol/ipa_prf/malicious_security/prover.rs

```diff
@@ -248,12 +252,12 @@ mod test {
             1, 1, 0, 0, 1, 1,
         ];
         const PROOF_1: [u128; 7] = [0, 30, 29, 30, 5, 28, 13];
-        const PROOF_LEFT_1: [u128; 7] = [1, 4, 10, 15, 12, 15, 29];
+        const PROOF_LEFT_1: [u128; 7] = [0, 11, 24, 8, 0, 4, 3];
         const U_2: [u128; 8] = [0, 0, 26, 0, 7, 18, 24, 13];
         const V_2: [u128; 8] = [10, 21, 30, 28, 15, 21, 3, 3];
 
         const PROOF_2: [u128; 7] = [12, 6, 15, 8, 29, 30, 6];
-        const PROOF_LEFT_2: [u128; 7] = [30, 28, 10, 25, 12, 23, 29];
+        const PROOF_LEFT_2: [u128; 7] = [5, 26, 14, 9, 0, 25, 2];
```

Comment:
LOL. Thanks for doing the work to find a new random 2-way sharing that generates the same value of `r` =).

Reply:
I hope we don't need to do it again :)